README.md

```diff
@@ -71,12 +89,22 @@ yarn lint --fix
 
 ### Running the tests
 
+#### Unit tests
+
 We uses [Jest](https://jestjs.io//) as the test engine. You can run unit tests by executing the following command:
 
 ```bash
 yarn test
 ```
 
+#### Functional tests
+
+```bash
+TEST_HEADLESS=false yarn test:cucumber
```

Comment:
Maybe explain what that env var does?

Reply:
👍 Good idea, I've added a note to make it clearer what this does in bfd8b64